app/templates/components/settings/contact-info-section.hbs

```diff
@@ -1,11 +1,11 @@
 <form class="ui form" {{action 'submit' on = 'submit'}}>
   <div class="field">
     <label>Email</label>
-    <input type="text" name="email" placeholder="Email">
+    {{input type="text" name="email" placeholder="Email"}}
```

Comment:
![value](https://cloud.githubusercontent.com/assets/17925669/26286036/2d94654c-3e79-11e7-8e58-d3f5b8a7b3ed.png)
Looks like this when I use value.